internal/cli/cmd/get/get.go

```diff
@@ -34,20 +38,30 @@ kargo get promotions --project=my-project --stage=my-stage
 `),
 	}
 
+	cmdOpts := &getOptions{}
+
 	// Register subcommands.
 	cmd.AddCommand(newGetCredentialsCommand(cfg, streams))
 	cmd.AddCommand(newGetFreightCommand(cfg, streams))
 	cmd.AddCommand(newGetProjectsCommand(cfg, streams))
 	cmd.AddCommand(newGetPromotionsCommand(cfg, streams))
 	cmd.AddCommand(newGetStagesCommand(cfg, streams))
 	cmd.AddCommand(newGetWarehousesCommand(cfg, streams))
+
+	cmdOpts.addFlags(cmd)
+
 	return cmd
 }
 
+func (o *getOptions) addFlags(cmd *cobra.Command) {
+	option.NoHeaders(cmd.PersistentFlags(), &o.NoHeaders)
```

Comment:
Ah. Good eye @hiddeco.

I like no. 2, personally. no. 1 would work, but I don't think I'd care for it being handled so much differently from `streams`. Consistency is a virtue.

Reply:
Yeah I like number two better as well. will sort that now

Reply:
@hiddeco this should resolve this:
https://github.com/akuity/kargo/pull/1777/commits/9bd37eeda943a857b1a2bf1dc9897387bffcca39.

And it is good that we did it, because I missed the flag registration on one resource so that would have been an annoying one considering we don't have tests for this.